test/parallel/test-cluster-disconnect-race.js

```diff
@@ -7,6 +7,12 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 const cluster = require('cluster');
+
+if (common.isWindows) {
+  console.log('1..0 # Skipped: This test is disabled on windows.');
```

Comment:
Yeah I just copied this directly from another test that presumably was doing a check for the same reason.